content/influxdb/v2.0/process-data/common-tasks/task-failure.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Notify on failed tasks
```

Comment:
Alert on failed tasks

(think alert is preferable bc we have that term on top level)